src/layer/poi.js

```diff
@@ -0,0 +1,103 @@
+import * as label from "../constants/label.js";
+import * as Color from "../constants/color.js";
+
+var iconDefs = {
+  bar: ["bar", "beer"],
+  hospital: "hospital",
+  medical: ["doctors", "clinic"],
+  parking: "parking",
+};
+
+export const poi = {
+  id: "poi",
+  type: "symbol",
+  paint: {
+    "text-halo-color": Color.backgroundFill,
+    "text-halo-width": 1.5,
+    "icon-halo-width": 0.4,
+    "text-halo-blur": 1,
+    "icon-halo-blur": 0.2,
+    "text-color": [
+      "match",
+      ["get", "class"],
+      ["bar", "beer"],
+      Color.poi.consumer,
+      ["hospital", "parking"],
+      Color.poi.infrastructure,
+      Color.poi.infrastructure,
+    ],
+  },
+  filter: [
+    ">=",
+    ["zoom"],
+    [
+      "match",
+      ["get", "subclass"],
+      "hospital",
+      15,
+      ["bar", "beer"],
+      16,
+      ["clinic", "doctors", "parking"],
+      17,
+      99,
+    ],
+  ],
+  layout: {
+    "text-font": ["Metropolis Light"],
```

Comment:
These labels should use the same font as the rest of the map to ensure better language coverage.

```suggestion
    "text-font": ["OpenHistorical"],
```